checker/check-api-added_test.go

```diff
@@ -0,0 +1,33 @@
+package checker_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tufin/oasdiff/checker"
+	"github.com/tufin/oasdiff/diff"
+)
+
+// BC: new paths or path operations are detected
```

Comment:
Comments begining with "// BC: ..." are used to generate the BREAKING-CHANGES-EXAMPLES.md doc.
See /scripts directory.

We need to decide how we want to document the INFO level checks.

Reply:
I like "// CL: ..."

Reply:
done for INFO level tests